Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -95,6 +95,11 @@ def customisePostEra_Run3_2023(process):
     customisePostEra_Run3(process)
     return process
 
+def customisePostEra_Run3_2024(process):
+    #start with a repeat of 2023
+    customisePostEra_Run3_2023(process)
```

Comment:
Maybe here could be a repeat of Run3 instead, with customizePostEra_Run3? This way it would not call the `run3_egamma_2023` modifier. Would it make sense?

Reply:
> This way it would not call the `run3_egamma_2023` modifier.

This is not the way to not call it. These are just copies of copies from Run-2.
If you want to avoid calling `run3_egamma_2023`  it needs to be done in the era definition itself (though it defies somewhat the purpose of the PR).